src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -39,7 +39,7 @@ public void testIsFull() {
 	@Test

```

Comment:
testIsFull is malformed: If STACK_CAPACITY is 12 then 12 elements should be added to the stack and it should be full.